working-mode

```diff
@@ -287,6 +287,56 @@ pragmatic solution in case of conflict.
 <p>Implementation disagreement should not result in implementation-defined behavior or optional
 features.
 
+<h3 id="anchors">Anchor permanence<a class="self-link" href="#anchors"></a></h3>
+
+<p>Often other standards want to reference parts of a WHATWG Living Standard by using a hypertext
+anchor. Since Living Standards are continually evolving, the set of anchors in a document is not
+static, and some anchors could disappear over time.
+
+<p>In general, editors should try to preserve anchors when reasonable. This includes using
+techniques like:
+
+<ul>
+  <li>For name changes, retaining the old anchor anyway, even if it doesn't match the new name.
+  (This might be in addition to adding a new anchor aligned with the new name.)
+
+  <li>For removals, adding a note explaining what was formerly there, and why it is gone, having the
+  old anchor now point to that note.
+</ul>
+
+<p class="example">The <code>console</code> object changed from being a Web IDL interface to a Web
+IDL namespace. Although the Console Standard's new anchor is
+<a href="https://console.spec.whatwg.org/#console-namespace">#console-namespace</a>, the old anchor
+of <a href="https://console.spec.whatwg.org/#console-interface">#console-interface</a> still
+works.</p>
+
+<p class="example">Although the concept of "structured cloning" has been replaced by structured
+serialization, the HTML Standard's
+<a href="https://html.spec.whatwg.org/multipage/structured-data.html#structured-clone">#structured-clone</a>
+anchor points to a note explaining this evolution, so that old links do not break.</p>
+
+<p>Consumers who need to reference exactly what a Living Standard said at a given point in time,
```

Comment:
Not sure if this paragraph belongs here; thoughts welcome.

Reply:
I think that's more what we want from others and not really how we do work. I'm not sure what a better place would be though so we might want to keep it here for now.